PCGen-base/code/src/java/pcgen/base/util/AbstractMapToList.java

```diff
@@ -335,11 +325,7 @@ public int sizeOfListFor(K key)
 		 * 24, 2005
 		 */
 		List<V> list = mapToList.get(key);
-		if (list == null)
-		{
-			return 0;
-		}
-		return list.size();
+		return (list == null) ? 0 : list.size();
```

Comment:
ditto, nullToEmpty (this is one of the many reasons I dislike nulls and prefer optional)